py/pipeline-workflow/check-draft-status.py

```diff
@@ -7,11 +7,35 @@
 with open("unicodetools/data/ucd/dev/DerivedAge.txt", 'r') as f:
     version = f.readline().strip().replace("# DerivedAge-", "").replace(".0.txt", "")
 
+PROVISIONALLY_ASSIGNED_AGE_NOTICE = f"""\f
+While the Unicode Technical Committee has provisionally assigned these
+characters, they have not been accepted for Unicode {version}, nor for any
+specific version of Unicode.
+
+The values of the Age property in this file are a placeholder.
+""".replace("\n", "%0A")
+
+UNREVIEWED_AGE_NOTICE = f"""\f
+These characters are neither accepted for Unicode {version}, nor for any
+specific version of Unicode, nor are they provisionally assigned.
+
+The values of the Age property in this file are a placeholder.
```

Comment:
Singular/plural mismatch.

And maybe better:
The Age property values for new characters are likely incorrect right now. They will be recomputed after the UTC accepts their encoding and this pull request is updated for the target version.